bazel/repository_locations.bzl

```diff
@@ -826,11 +826,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WAVM",
         project_desc = "WebAssembly Virtual Machine",
         project_url = "https://wavm.github.io",
-        version = "c8997ebf154f3b42e688e670a7d0fa045b7a32a0",
-        sha256 = "bf2b2aec8a4c6a5413081c0527cb40dd16cb67e9c74a91f8a82fe1cf27a3c5d5",
-        strip_prefix = "WAVM-{version}",
+        version = "nightly/2022-05-14",
+        sha256 = "7cfa3d7334c96f89553bb44eeee736a192826a78b4db114042d38d6882748f5b",
+        strip_prefix = "WAVM-nightly-2022-05-14",
```

Comment:
I couldn't figure out how to make this work with `{version}`:
- `"WAVM-{version}".replace("/", "-")` doesn't work (replacement is performed before variable evaluation),
- the dependency check script complains about unknown release `2022-05-14` when using `nightly` in `strip_prefix` and `urls` instead of `version`.

I'm open to suggestions if current approach is not acceptable.

Reply:
Actually, it doesn't work even with `nightly/2022-05-14`:
```
DependencyChecker ERROR [releases] Check disabled, `preload_releases` failed: No commit found for SHA: 2022-05-14
DependencyChecker ERROR [release_issues] Check disabled, `preload_releases` failed: No commit found for SHA: 2022-05-14
DependencyChecker ERROR [release_dates] Check disabled, `preload_releases` failed: No commit found for SHA: 2022-05-14
```
@phlax any ideas how to fix it?

Reply:
not sure off hand - the script is a little brittle for parsing these - i can take a look tomorrow

Reply:
OK, I'll change to commit SHA to unblock it in the meantime.